ExtraDry/ExtraDry.Core/ExpandoData/ExpandoValues.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Text.Json;
+
+namespace ExtraDry.Core; 
+
+[JsonConverter(typeof(ExpandoValuesConverter))]
+public class ExpandoValues : IValidatableObject {
```

Comment:
Pasq did something similar, a class that needed a converter.  He ended up doing it a different way because the Swagger documentation got hosed by the JSON Converter.  Perhaps this is just a `Dictionary<string, object>` and not a named object?  Regardless, would be nice if the two approaches could be aligned.

Reply:
This has been removed.